src/makeCalculator.js

```diff
@@ -37,7 +37,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    operate(callback, value) {
+      if (!isNaN(value)) {
+        this.result = callback(value, this.result);
+
+        return this;
+      }
+    },
+    add(value, result) {
+      return result + value;
+    },
+    subtract(value, result) {
+      return result - value;
+    },
+    divide(value, result) {
+      if (value !== 0) {
```

Comment:
If 'value' is zero, the divide function will not return anything, leading to undefined behavior. It's better to throw an error or return a specific value.